dll/win32/shell32/shellrecyclebin/recyclebin_v5_enumerator.c

```diff
@@ -34,6 +41,14 @@ RecycleBin5File_RecycleBinFile_QueryInterface(
         *ppvObject = &s->recycleBinFileImpl;
     else if (IsEqualIID(riid, &IID_IRecycleBinFile))
         *ppvObject = &s->recycleBinFileImpl;
+    else if (IsEqualIID(riid, &IID_IExtractIconA) || IsEqualIID(riid, &IID_IExtractIconW))
+    {
+        DWORD dwAttributes;
+        if (RecycleBin5File_RecycleBinFile_GetAttributes(This, &dwAttributes) == S_OK)
```

Comment:
S_OK, otherwise we use `SUCCEEDED` in case there could be other returned values that are still successful.